h/feeds/rss.py

```diff
@@ -1,4 +1,8 @@
 """Functions for generating RSS feeds."""
+
+from calendar import timegm
+from email.utils import formatdate
```

Comment:
Great use of the standard library. I'm also learning about new standard lib things from your PRs!